source/_lovelace/horizontal-stack.markdown

```diff
@@ -7,7 +7,11 @@ description: "The Horizontal Stack card allows you to stack together multiple ca
 
 The Horizontal Stack card allows you to stack together multiple cards, so they always sit next to each other in the space of one column.
 
-To add the Horizontal Stack card to your user interface, click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**. Click the "Add Card" button in the bottom right corner and select **Horizontal Stack** from the card picker. All options for this card can be configured via the user interface.
+To add the Horizontal Stack card to your user interface, click the Lovelace menu (three dots at the top right of the screen) and then **Edit Dashboard**. Click the "Add Card" button in the bottom right corner and select **Horizontal Stack** from the card picker.
+
+## YAML
+
+This is for if you use YAML mode or just prefer to use YAML in the Code Editor in the UI.
```

Comment:
I kinda dig this idea, to make more clear what the configuration section is, however, should we apply this consistently? This PR only adds it in 2 places.

Also, we could make the title a little more explanatory. For example:

```suggestion
## YAML Configuration

The following YAML options are available when you use YAML mode or just prefer to use YAML in the Code Editor in the UI:
```

Reply:
I think it would be great if they were more consistent. Should I update them all in this PR? I would prefer to go through them all in a separate PR and I can try to make them all more similar.

Reply:
Yeah, sure, if you want to help out standardize that part, that would be awesome! ❤️

Reply:
I have now spent some time going through the pages for all the cards. They now look the same in regards to headers, I believe this is a lot better.

Note that I haven't added any missing descriptions. This means that some pages are missing explanation for some UI fields. But they all look the same know, which I think is a big improvement

Some other things I changed

* I also removed the exemption parts from button and picture that are moved to the actions page since a long time ago, see https://github.com/home-assistant/home-assistant.io/pull/12229
* I moved the text how to add cards from the page describing each card to the main page. I don't think it is necessary to repeat so many times at the top of all the sub pages.

Reply:
I have added back this text, please take another look.